API/StoryAPI.cs

```diff
@@ -166,6 +166,30 @@ public static void Map(WebApplication app)
                 }
                 return Results.Ok(searchResults);
             });
+
+            //GET STORIES BY CATEGORY
+            app.MapGet("/stories/categories/{categoryId}", (FanFusionDbContext db, int categoryId) =>
+            {
+                var category = db.Categories.FirstOrDefault(c => c.Id == categoryId);
+
+                // Check if the category exists
+                if (category == null)
+                {
+                    return Results.NotFound($"Category with ID {categoryId} not found.");
+                }
+
+                var stories = db.Stories
+                    .Include(s => s.Category)
```

Comment:
You can take this line out actually because we aren't using the category object and we are just grabbing the categoryId which is available in the stories model.